AndriiDryha/HW7/PRACTICAL TASKS [LESSON 7].py

```diff
@@ -0,0 +1,56 @@
+# # Task 1
+
+# def largest_number(n, m):
+        
+#         """This function returns largest number of two numbers"""
+        
+#         if n > m:
+#             return n
+#         else:
+#             return m
+        
+# result = largest_number(int(input("Enter first number \n")), int(input("Enter second number \n")))
+
+# print("The largest number of two entered numbers:", result)
+
+
+# # Task 2
+
+# Choice = input("Write the area of which figure you want to calculate: 'rectangle', 'triangle' or 'circle' \n")
+
+# while True:
+
+#     if Choice == "rectangle":
+#         import Rectangle
+#         print(Rectangle.rectangle_area)
+#         break
+#     if Choice == "triangle":
+#         import Triangle
+#         print(Triangle.triangle_area)
+#         break
+#     if Choice == "circle":
+#         import Circle
+#         print(Circle.circle_area)
+#         break
+
+# Task 3
+
+def number_character(word):
```

Comment:
good job
look at this version
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result